### PR TITLE
Update omnibus-software to pick up chef/omnibus-software#1451 and chef/omnibus-software#1453.

### PR DESCRIPTION
**Pull request by davidbtucker**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/chef/omnibus-software.git
-  revision: bce0974a4f64bce2cc100a5baae26031ea430611
+  revision: 282c422ddb8b8e4af9d2f1cb8011a280e63f5308
```
Comment (igorpeshansky) [inline Gemfile.lock:3]: Just wondering, do we need chef/omnibus-software#1453 as well (commit [`a901ec26f8c704ba793c0609319f00f49e630d6b`](https://github.com/chef/omnibus-software/commit/a901ec26f8c704ba793c0609319f00f49e630d6b))?

**Review by igorpeshansky — COMMENTED** [review]
Note: there was a typo in the commit message that was propagated into the PR title. I've fixed the PR title, but I think GitHub will keep the commit message if there's only one, so watch out for that.

File: Gemfile.lock (hunk 1 above)
Comment (davidbtucker) [reply Gemfile.lock]: Done!

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Pull request merged**